webviz_subsurface/plugins/_co2_leakage/_plugin.py

```diff
@@ -386,9 +395,7 @@ def make_unit_list(
             Output(ViewSettings.Ids.WELL_FILTER_HEADER, "style"),
             Input(self._settings_component(ViewSettings.Ids.ENSEMBLE), "value"),
         )
-        def set_well_options(
-            ensemble: str,
-        ) -> Tuple[List[Any], List[str], Dict[Any, Any], Dict[Any, Any]]:
+        def set_well_options(ensemble: str) -> Any:
```

Comment:
For å fikse noe pylint-klaging eller noe?

Reply:
Var noe mypy-styr, ja. Kan eventuelt prøve å få det helt rett.

Reply:
Kan prøve hvis det ikke tar for lang tid